<?xml version="1.0" encoding="utf-8" standalone="no"?>
<!--This file represents the results of running a test suite-->
<test-results name="U:\Data\Data\Programming\Projects\NET_TestHarness\TestCases\TestCases.csproj" total="36" errors="0" failures="0" not-run="0" inconclusive="0" ignored="0" skipped="0" invalid="0" date="2012-06-28" time="18:48:35">
  <environment nunit-version="2.6.0.12051" clr-version="2.0.50727.5448" os-version="Microsoft Windows NT 6.1.7601 Service Pack 1" platform="Win32NT" cwd="U:\Data\Data\Programming\Projects\NET_TestHarness\TestCases" machine-name="MAIN-PC" user="Michael" user-domain="MAIN-PC" />
  <culture-info current-culture="en-US" current-uiculture="en-US" />
  <test-suite type="Project" name="U:\Data\Data\Programming\Projects\NET_TestHarness\TestCases\TestCases.csproj" executed="True" result="Success" success="True" time="0.499" asserts="0">
    <results>
      <test-suite type="Assembly" name="U:\Data\Data\Programming\Projects\NET_TestHarness\TestCases\bin\Debug\TestCases.dll" executed="True" result="Success" success="True" time="0.491" asserts="0">
        <results>
          <test-suite type="Namespace" name="TestCases" executed="True" result="Success" success="True" time="0.491" asserts="0">
            <results>
              <test-suite type="Namespace" name="ChkUtilsTests" executed="True" result="Success" success="True" time="0.491" asserts="0">
                <results>
                  <test-suite type="TestFixture" name="ValidatorTests" executed="True" result="Success" success="True" time="0.477" asserts="0">
                    <results>
                      <test-case name="TestCases.ChkUtilsTests.ValidatorTests.Disposed_Fail" executed="True" result="Success" success="True" time="0.298" asserts="5" />
                      <test-case name="TestCases.ChkUtilsTests.ValidatorTests.Disposed_Fail_FaultException" executed="True" result="Success" success="True" time="0.048" asserts="5" />
                      <test-case name="TestCases.ChkUtilsTests.ValidatorTests.Disposed_Valid" executed="True" result="Success" success="True" time="0.001" asserts="1" />
                      <test-case name="TestCases.ChkUtilsTests.ValidatorTests.Disposed_Valid_FaultException" executed="True" result="Success" success="True" time="0.000" asserts="1" />
                      <test-case name="TestCases.ChkUtilsTests.ValidatorTests.ExceptionType_Fault_False" executed="True" result="Success" success="True" time="0.001" asserts="0" />
                      <test-case name="TestCases.ChkUtilsTests.ValidatorTests.ExceptionType_Fault_Param" executed="True" result="Success" success="True" time="0.000" asserts="0" />
                      <test-case name="TestCases.ChkUtilsTests.ValidatorTests.ExceptionType_Fault_StringEmpty" executed="True" result="Success" success="True" time="0.002" asserts="0" />
                      <test-case name="TestCases.ChkUtilsTests.ValidatorTests.ExceptionType_Fault_StringNull" executed="True" result="Success" success="True" time="0.000" asserts="0" />
                      <test-case name="TestCases.ChkUtilsTests.ValidatorTests.ExceptionType_Fault_True" executed="True" result="Success" success="True" time="0.000" asserts="0" />
                      <test-case name="TestCases.ChkUtilsTests.ValidatorTests.ExceptionType_Fault_Var" executed="True" result="Success" success="True" time="0.001" asserts="0" />
                      <test-case name="TestCases.ChkUtilsTests.ValidatorTests.ExceptionType_Regular_False" executed="True" result="Success" success="True" time="0.000" asserts="0" />
                      <test-case name="TestCases.ChkUtilsTests.ValidatorTests.ExceptionType_Regular_Param" executed="True" result="Success" success="True" time="0.001" asserts="0" />
                      <test-case name="TestCases.ChkUtilsTests.ValidatorTests.ExceptionType_Regular_String_Empty" executed="True" result="Success" success="True" time="0.000" asserts="0" />
                      <test-case name="TestCases.ChkUtilsTests.ValidatorTests.ExceptionType_Regular_String_Null" executed="True" result="Success" success="True" time="0.000" asserts="0" />
                      <test-case name="TestCases.ChkUtilsTests.ValidatorTests.ExceptionType_Regular_True" executed="True" result="Success" success="True" time="0.001" asserts="0" />
                      <test-case name="TestCases.ChkUtilsTests.ValidatorTests.ExceptionType_Regular_Var" executed="True" result="Success" success="True" time="0.001" asserts="0" />
                      <test-case name="TestCases.ChkUtilsTests.ValidatorTests.False_Fail" executed="True" result="Success" success="True" time="0.000" asserts="5" />
                      <test-case name="TestCases.ChkUtilsTests.ValidatorTests.False_Fail_FaultException" executed="True" result="Success" success="True" time="0.000" asserts="5" />
                      <test-case name="TestCases.ChkUtilsTests.ValidatorTests.False_Valid" executed="True" result="Success" success="True" time="0.000" asserts="1" />
                      <test-case name="TestCases.ChkUtilsTests.ValidatorTests.False_Valid_FaultException" executed="True" result="Success" success="True" time="0.000" asserts="1" />
                      <test-case name="TestCases.ChkUtilsTests.ValidatorTests.Param_NullArg" executed="True" result="Success" success="True" time="0.001" asserts="5" />
                      <test-case name="TestCases.ChkUtilsTests.ValidatorTests.Param_NullArg_FaultException" executed="True" result="Success" success="True" time="0.000" asserts="5" />
                      <test-case name="TestCases.ChkUtilsTests.ValidatorTests.Param_ValidArg" executed="True" result="Success" success="True" time="0.000" asserts="1" />
                      <test-case name="TestCases.ChkUtilsTests.ValidatorTests.Param_ValidArg_FaultException" executed="True" result="Success" success="True" time="0.001" asserts="1" />
                      <test-case name="TestCases.ChkUtilsTests.ValidatorTests.String_Empty" executed="True" result="Success" success="True" time="0.000" asserts="5" />
                      <test-case name="TestCases.ChkUtilsTests.ValidatorTests.String_Empty_FaultException" executed="True" result="Success" success="True" time="0.001" asserts="5" />
                      <test-case name="TestCases.ChkUtilsTests.ValidatorTests.String_Null" executed="True" result="Success" success="True" time="0.001" asserts="5" />
                      <test-case name="TestCases.ChkUtilsTests.ValidatorTests.String_Null_FaultException" executed="True" result="Success" success="True" time="0.001" asserts="5" />
                      <test-case name="TestCases.ChkUtilsTests.ValidatorTests.True_Fail" executed="True" result="Success" success="True" time="0.000" asserts="5" />
                      <test-case name="TestCases.ChkUtilsTests.ValidatorTests.True_Fail_FaultException" executed="True" result="Success" success="True" time="0.000" asserts="5" />
                      <test-case name="TestCases.ChkUtilsTests.ValidatorTests.True_Valid" executed="True" result="Success" success="True" time="0.001" asserts="1" />
                      <test-case name="TestCases.ChkUtilsTests.ValidatorTests.True_Valid_FaultException" executed="True" result="Success" success="True" time="0.000" asserts="1" />
                      <test-case name="TestCases.ChkUtilsTests.ValidatorTests.Var_NullArg" executed="True" result="Success" success="True" time="0.001" asserts="5" />
                      <test-case name="TestCases.ChkUtilsTests.ValidatorTests.Var_NullArg_FaultException" executed="True" result="Success" success="True" time="0.001" asserts="5" />
                      <test-case name="TestCases.ChkUtilsTests.ValidatorTests.Var_ValidArg" executed="True" result="Success" success="True" time="0.000" asserts="1" />
                      <test-case name="TestCases.ChkUtilsTests.ValidatorTests.Var_ValidArg_FaultException" executed="True" result="Success" success="True" time="0.000" asserts="1" />
                    </results>
                  </test-suite>
                </results>
              </test-suite>
            </results>
          </test-suite>
        </results>
      </test-suite>
    </results>
  </test-suite>
</test-results>